enh(release): Add Centreon Open Tickets 21.10.1 RN (#1674)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-os-extensions.md

## Centreon Open Tickets

### 21.10.1

#### Enhancements

- [Core] Added PHP 8.0 compatibility
- [Core] Added ability to use user name in smarty configuration for providers
- [Core] Use options defined in administration menu for acknowledgement
- [Install] Added ability to use a non default database name
- [Widget] Added an alarm duration filter in widget configuration
- [Widget] Added an option to schedule a check when you open a ticket or acknowledge a resource
- [Widget] Preselect option if only one choice is available

#### Bug fixes

- [Provider] Fixed error 400 issue when opening a ticket using Jira provider
- [Widget] Display of the state type of the host instead of the service

> 

Reviewer: Missing ticket : https://centreon.atlassian.net/browse/MON-12704
[open-tickets] Regression Mail provider

## Centreon Open Tickets

### 21.10.1

#### Enhancements

- [Core] Added PHP 8.0 compatibility

> - [Core] Added ability to use user name in smarty configuration for providers

Reviewer 2: ```suggestion
- [Core] Added the ability to use user name when using Smarty configuration for providers
```

> - [Core] Use options defined in administration menu for acknowledgement

Reviewer 2: 'Added defined options for acknowledgement in the administration menu'?

- [Install] Added ability to use a non default database name
- [Widget] Added an alarm duration filter in widget configuration
- [Widget] Added an option to schedule a check when you open a ticket or acknowledge a resource

> - [Widget] Added preselect option in select box if only one choice is available

Reviewer 2: ```suggestion
- [Widget] Added a preselect option in the select box if only one choice is available
```

#### Bug fixes

> - [Provider] Fixed error 400 issue when opening a ticket using Jira provider

Reviewer 2: ```suggestion
- [Provider] Fixed the error 400 issue when opening a ticket using Jira provider
```

> - [Provider] Fixed regression with Mail provider

Reviewer 2: ```suggestion
- [Provider] Fixed a regression with the Mail provider
```

> - [Widget] Display of the state type of the host instead of the service

Reviewer 2: Should it be "Fixed the state type that displayed the host instead of the service"?

Author: no, previously the column state display service value instead of host value.
